lib/client/socket.js

```diff
@@ -26,6 +19,15 @@ module.exports = ({ url, token, filters, config, identifyingMetadata }) => {
     throw error;
   }
 
+  const Socket = Primus.createSocket({
+    transformer: 'engine.io',
+    parser: 'EJSON',
+    plugin: {
+      'emitter': require('primus-emitter'),
+    },
+    pathname : '/primus/'+token,
```

Comment:
In the moment this defaults to just /primus (see: https://github.com/primus/primus/blob/master/index.js#L67) but the server will accept anything under /primus. The token can be picked up by a load balancer to route the request. See: https://github.com/snyk/ops/blob/master/helm/broker-container-server/templates/load-balancing.yaml#L13

Reply:
What are we going to load balance? Isn't it stateless thus current round-robin should be enough? Maybe I'm missing some context for this PR.

Reply:
The broker is not stateless as Agent and Broker Client need to access the same server instance to be able to communicate.

Reply:
```suggestion
    pathname: `/primus/${token}`,
```